docs/source/migrate_to_notebook7.rst

```diff
@@ -0,0 +1,73 @@
+.. _htmlnotebook:
+
+NbClassic and Notebook
```

Comment:
Since this will be part of the Notebook docs, I'm wondering whether the title should say something like "Updating to Notebook 7" instead?

Some Notebook users might not know what NBClassic is.

Especially since it will be visible in the navigation bar at the top of the page:

![image](https://user-images.githubusercontent.com/591645/219590903-1604d3de-015b-412e-bc7a-b71e9716afd5.png)

Reply:
Which means there could be a first section on this page summarizing [JEP 79](https://jupyter.org/enhancement-proposals/79-notebook-v7/notebook-v7.html) with new features, the change of the extension system, switch to jupyter server.

And maybe the sections below should be called "Compatibility with older versions" or "Using the Classic Notebook UI".

Reply:
Thx @jtpio I have update the migrate_to_notebook7.rst page based on your feedbacks. The idea here is to bootstrap that page and further enrich it.